include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -630,11 +659,28 @@ MultigridPreconditionerBase<dim, Number>::initialize_affine_constraints(
     boundary_functions[it.first] = &zero_function;
   }
 
-  dealii::MappingQ<dim> mapping_dummy(1);
-  dealii::VectorTools::interpolate_boundary_values(mapping_dummy,
-                                                   dof_handler,
-                                                   boundary_functions,
-                                                   affine_constraints);
+
+  if(triangulation->all_reference_cells_are_hyper_cube())
+  {
+    dealii::MappingQ<dim> mapping_dummy(1);
```

Comment:
https://www.dealii.org/developer/doxygen/deal.II/classReferenceCell.html#adbfa6390e481aca723b8086db8b7d12c

Reply:
Addressed in 60211edbe758f3ed0688ae53da0c7410a3104d79.